src/storage.ts

```diff
@@ -260,16 +258,57 @@ export class Storage {
     }
   }
 
-  private binaryPath(version: string): string {
+  private binaryPath(): string {
     const os = goos()
     const arch = goarch()
-    let binPath = path.join(this.getBinaryCachePath(), `coder-${os}-${arch}-${version}`)
+    let binPath = path.join(this.getBinaryCachePath(), `coder-${os}-${arch}`)
```

Comment:
🔨  We no longer use the version in the binaryPath. This should fix the Windows prompt issue.